src/init/command.template

```diff
@@ -7,5 +7,26 @@
 cd { host_shared }
 {{ endif }}
 
+# Discover where the output chardev is located
```

Comment:
Thinking about pulling out vport discovery into qemu.rs. So that we can generate better error messages. `run_in_vm()` could be called with a sink that just appends output to a string. And we can parse that output for any discovered vport.